src/components/Notification/icons/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import InfoIcon from './infoIcon';
+import ErrorIcon from './errorIcon';
+import SuccessIcon from './successIcon';
+import WarningIcon from './warningIcon';
+import '../styles.css';
```

Comment:
you don't need to import the css file again if it is already imported in index of Notification